src/engine/source/kvdb2/include/kvdb2/kvdbManager.hpp

```diff
@@ -0,0 +1,91 @@
+#ifndef _KVDBMANAGER_H
```

Comment:
I think it should be `_KVDB_MANAGER_H`

Reply:
Will be fixed in later documentation step.